flask_app/blueprints/api/errors.py

```diff
@@ -20,8 +20,8 @@
 NoneType = type(None)
 
 
-@API(version=4)
-def add_error(message: str, exception_type: (str, NoneType)=None, traceback: (list, NoneType)=None, timestamp: (float, int)=None, test_id: int=None, session_id: int=None, is_failure: bool=False, is_interruption: bool=False): # pylint: disable=bad-whitespace
+@API(version=5)
+def add_error(message: str, exception_type: (str, NoneType)=None, traceback: (list, NoneType)=None, timestamp: (float, int)=None, test_id: int=None, session_id: int=None, is_failure: bool=False, is_interruption: bool=False, is_fatal: bool=False): # pylint: disable=bad-whitespace
```

Comment:
@maormarcus The default should not be False (in case someone uses an old backslash-python package which doesn't report it.
In the API it would be important to have the ability to know if the error is not-fatal or just wasn't reported with this flag at all.